src/js/hooks/outboundImport/useOutboundImportForm.js

```diff
@@ -0,0 +1,53 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import moment from 'moment/moment';
+import { useForm } from 'react-hook-form';
+
+import { DateFormat } from 'consts/timeFormat';
+import useOutboundImportValidation from 'hooks/outboundImport/useOutboundImportValidation';
+
+const useOutboundImportForm = ({ next }) => {
+  const { validationSchema } = useOutboundImportValidation();
+  const getDefaultValues = () => ({
+    description: undefined,
+    origin: undefined,
+    destination: undefined,
+    requestedBy: undefined,
+    dateRequested: undefined,
+    dateShipped: moment(new Date()).format(DateFormat.MMM_DD_YYYY_HH_MM_SS),
+    shipmentType: undefined,
+    trackingNumber: undefined,
+    comments: undefined,
+    expectedDeliveryDate: undefined,
+    packingList: undefined,
```

Comment:
Are those `undefined` necessary to use these values as defaults?

Reply:
For now yeah, the eventual adjustments (if we will even support the "edit" functionality for this feature), should be dealt with in the "integrate backend & frontend" tucket.